app/customer.py

```diff
@@ -0,0 +1,43 @@
+import dataclasses
+import datetime
+from typing import Any
+
+
+@dataclasses.dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list[int]
+    money: float
+    car: dict[str, [str, float]]
+
+    @classmethod
+    def load_from_dict(cls, customer: dict[str, Any]) -> None:
+        cls.name = customer["name"]
+        cls.product_cart = customer["product_cart"]
+        cls.location = customer["location"]
+        cls.money = customer["money"]
+        cls.car = customer["car"]
```

Comment:
We can not do cls.something cls is a type
return cls(
   # And here all property
)